content/docs/examples/shared-resources.md

```diff
@@ -0,0 +1,33 @@
++++
+title = "Shared Resources and Components"
+description = "Hubs where you can find reusable components, shared Jupyter notebooks, and other ML resources"
+weight = 50
++++
+
+This page links to websites where you can find machine learning 
+(ML) resources shared by various communities and organizations.
+
+## AI Hub
+
+[AI Hub](https://aihub.cloud.google.com/) is a platform for discovering and 
+deploying ML products.
+
+AI Hub includes the following shared resources that you can use within your 
+Kubeflow deployment:
+
+* [Pipelines and components](https://aihub.cloud.google.com/s?category=pipeline) 
+  that you can use with Kubeflow Pipelines.
+* [Jupyter notebooks](https://aihub.cloud.google.com/s?category=notebook) that 
+  you can upload to the notebooks server in your Kubeflow cluster.
+
+## Reusable components for Kubeflow Pipelines
+
+A Kubeflow Pipelines *component* is a self-contained set of code that performs 
+one step in the pipeline, such as data preprocessing, data transformation, model
+training, and so on. Each component is packaged as a Docker image.
```

Comment:
Would it make sense to warn users to only use components from repositories that they trust? Since the executable code in a component is packaged as a docker image, the user needs to trust that the component does what it says it does or they need to investigate each image to verify that they are safe.

Reply:
Done. PTAL.